src/library/basetracktablemodel.cpp

```diff
@@ -463,6 +467,40 @@ bool BaseTrackTableModel::setData(
     return setTrackValueForColumn(pTrack, column, value, role);
 }
 
+QVariant BaseTrackTableModel::composeCoverArtToolTipHtml(
+        const QModelIndex& index) const {
+    // Determine height of the cover art image depending on the screen size
+    unsigned int absHeightOfCoverartTooltip;
+    const QScreen* primaryScreen = getPrimaryScreen();
```

Comment:
The primary screen is the wrong place. We need to wait for #3215 that allows to determine the screen of a QWidget.

Reply:
I'm waiting for this. But how can I determine the QWidget in this function? Isn't the table widget seperated from the library here?

Reply:
Oh, you are right, we don't have access to any widget here :thinking:

Reply:
Keep using getPrimaryScreen() here. Sorry for the confusion.